client/app/bundles/course/admin/pages/CodaveriSettings/components/AssessmentList.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC } from 'react';
+import { List } from '@mui/material';
+import { AssessmentProgrammingQuestionsData } from 'types/course/admin/codaveri';
+
+import Section from 'lib/components/core/layouts/Section';
+import Subsection from 'lib/components/core/layouts/Subsection';
+import useItems from 'lib/hooks/items/useItems';
+import { useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import { getAllAssessments } from '../selectors';
+import translations from '../translations';
+
+import AssessmentListItem from './AssessmentListItem';
+import CodaveriEnableDisableButtons from './CodaveriEnableDisableButtons';
+
+export const sortFunc = (
+  assessments: AssessmentProgrammingQuestionsData[],
+): AssessmentProgrammingQuestionsData[] => {
+  const sortedAssessments = [...assessments];
+  sortedAssessments.sort((a, b) =>
+    a.title.toLowerCase().trim() <= b.title.toLowerCase().trim() ? -1 : 1,
+  );
+  return sortedAssessments;
+};
+
+const AssessmentList: FC = () => {
+  const assessmentsWithProgrammingQn = useAppSelector((state) =>
+    getAllAssessments(state),
+  );
+  const { processedItems: sortedAssessments } = useItems(
+    assessmentsWithProgrammingQn,
+    [],
+    sortFunc,
+  );
+  const { t } = useTranslation();
+  return (
+    <Section
+      contentClassName="flex flex-col space-y-3"
+      sticksToNavbar
+      subtitle={t(translations.programmingQuestionSettingsSubtitle)}
+      title={t(translations.programmingQuestionSettings)}
+    >
+      <Subsection
+        title={t(translations.assessments)}
+        titleAction={<CodaveriEnableDisableButtons />}
+      >
+        <List
+          className="w-full border border-solid border-neutral-300 rounded-lg"
+          component="nav"
```

Comment:
Same here, I was following MUI example. Should it be div here?

Reply:
I'm not sure which MUI example you're referring to here, but if they do, I believe it's because the example they give is an actual navigation menu, e.g., sidebar, drawer, etc. In that case, using `nav` would be semantically correct.

In this case, we are using `List` just as a plain list. I think we can just remove the `component` prop altogether and let it render the default element.